package.json

```diff
@@ -195,6 +195,7 @@
     "makelogs": "3.0.2",
     "marked-text-renderer": "0.1.0",
     "mocha": "2.5.3",
+    "murmurhash3js": "3.0.1",
```

Comment:
We're using sha256 for the state pr, because sha256 provides better unique-ness. Here we're using murmur3 because the hashes it produces are numbers, making it trivial to bucket based on the hash (see [get_shard_num.js](https://github.com/spalger/kibana/blob/test/shardTestsToEaseMemoryPressure/src/ui/public/test_harness/test_sharding/get_shard_num.js)).

So, we could, but I think the problems are different and warrant different hashing algorithms.